abb_irb6640_support/urdf/irb6640_185_280_macro.xacro

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+<xacro:macro name="abb_irb6640_185_280" params="prefix">
+    <!-- link list -->
+    <link name="${prefix}base_link">
+        <collision name="collision">
+            <geometry>
+                <mesh filename="package://abb_irb6640_support/meshes/irb6640_185_280/collision/base_link.stl"/>
+            </geometry>
+            <material name="yellow">
+                <color rgba="0 1 1 1"/>
+            </material>
```

Comment:
If we move this pkg to `abb_resources`, we could use the new material macros you introduced.